pagopa/pn-frontend-e2e
# PN-9254-send-a-notification-to-a-multi-recipient-duplicated

## Short description

Created new test: PA try to send a notification to a multi-recipient duplicated

## List of changes proposed in this pull request

- Created new test: PA try to send a notification to a multi-recipient duplicated, then visualize an error due to the duplicated fiscal code
- Implemented a general step to insert value in the "destinatario" fields by number of it
- modified step to check the error in the field "codice fiscale"

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@TA_invioNotificaMultidestinatarioDuplicato"` and verify if the test works fine.
